src/Driver.php

```diff
@@ -7,23 +7,30 @@
 use Doctrine\DBAL\Driver\Exception;
 use Doctrine\DBAL\Platforms\AbstractPlatform;
 use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use SensitiveParameter;
 
 /**
  * Driver interface.
  * Interface that all DBAL drivers must implement.
+ *
+ * @psalm-import-type Params from DriverManager
```

Comment:
Should static analysis stuff be in a separate commit?

Reply:
Do you want me to split this off? I was merely fixing some inconsistencies on the go.

Reply:
Just did it for you :slightly_smiling_face: